sdk/tables/azure-data-tables/tests/test_table_entity.py

```diff
@@ -282,6 +282,25 @@ def _assert_merged_entity(self, entity):
         # self.assertIsNotNone(entity.Timestamp)
         # self.assertIsInstance(entity.Timestamp, datetime)
 
+    def _assert_valid_full_metadata(self, metadata):
+        keys = metadata.keys()
+        self.assertIn("client_request_id", keys)
+        self.assertIn("request_id", keys)
+        self.assertIn("version", keys)
+        self.assertIn("date", keys)
+        self.assertIn("etag", keys)
+        self.assertIn("preference_applied", keys)
+        self.assertIn("content_type", keys)
```

Comment:
We don't need all this information... lets start by filtering the list to just `date`, `etag`,`version`. Then if we need to add more things later we can do that.